README.md

```diff
@@ -37,8 +40,26 @@ This action deletes versions of a package from [GitHub Packages](https://github.
 
   # The number of old versions to delete starting from the oldest version.
   # Defaults to 1.
+  # Cannot be more than 100
   num-old-versions-to-delete:
 
+  # The number of latest versions to not delete.
+  # Defaults to 0.
+  # When this is set greater than 0 it will delete all deletable package versions except the specified no.
+  # `num-old-versions-to-delete` will not be taken into account.
+  # Cannot be more than 100
+  min-versions-to-keep:
+
+  # The package versions to ignore exclude from deletion.
```

Comment:
can we update the text to suggest that we are supporting regex here.

Reply:
Sure will make the change.